config/routes.js

```diff
@@ -96,4 +97,10 @@ module.exports = function(app, passport, auth) {
     app.get('/play', index.play);
     app.get('/', index.render);
 
-};
\ No newline at end of file
+    // Invitation
+    var middleWare = require('../config/middlewares/isLoggedIn')
```

Comment:
Let's move this to the top of the file @OlatunjiYso @Rhotimee

Reply:
Ooh. I'd do that now.